### ISSUE TITLE
Not an issue but valuable Feedback

### ISSUE THREAD

**Issue by advanivivek** [flukeout/restaurant#196]
### Would be awesome if you could implement this:

**When the user enters some CSS that is not the correct answer, you could still highlight the items being selected with user's CSS.**

That way, a person can learn more about the actual concepts and figure out where he/she is going wrong instead of just aiming for the right answer (or just trying out everything only to go forward).

Thank you :) Wonderful project. Thoroughly enjoyed it!

**Comment by esaramago** [comment]
It does that. When you select an element wrongly, it shakes.

**Comment by advanivivek** [comment]
Oh dear! I feel stupid now haha 😁 Thank you for the update.

**Comment by bill-richards** [comment]
@esaramago 

> It does that. When you select an element wrongly, it shakes.

yes, it shakes, however the OP said that _showing what the CSS selector actually selects would be very useful_, and he is not wrong.

**Comment by advanivivek** [comment]
Sweet haha ✌️🙏❤️🎶✨

>
